File: C/day3.c
#include <stdio.h>

int main(){
    int N; 
    scanf("%d", &N);
    if (N % 2 == 1 || (N % 2 == 0 && N <=20 && N >= 6)) { 
        printf("Weird");
    } else if (N % 2 == 0 && N <= 5 && N >= 2 || N > 20) {
        printf("Not Weird");
    }
    return 0;
}

File: JAVASCRIPT/day3.js
function main() {
    var N = parseInt(readLine());
    var output;

    if (N % 2 == 0) {
        if (N >= 2 && N <= 5) {
            output = "Not Weird";
        } else if (N >= 6 && N <= 20) {
            output = "Weird";
        } else {
            output = "Not Weird";
        }
    } else {
        output = "Weird";
    }

    console.log(output);
}
